src/filesystem.cpp

```diff
@@ -18,6 +18,7 @@
 #ifdef _MSC_VER
 #   include "wdirent.h"
 #   include <direct.h>
+#   define chdir _chdir
```

Comment:
This definition for MSVC is there to workaround the fact that posix (`chdir`) and windows (`_chdir`) directory change functions are not named the same. So this `#define` creates an alias under windows, which is used later in the code.
I've been referring to this [answer on SO](http://stackoverflow.com/a/40230786)  when introducing this change